package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-create-issue",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
Don't bump the version. I do this on my end before publishing to ensure everything gets properly tagged, etc.